bot/exts/filters/filtering.py

```diff
@@ -400,6 +473,8 @@ async def _send_log(
         stats: Stats,
         reason: Optional[str] = None,
         *,
+        ping: Optional[bool] = None,
+        message_override: str = "",
```

Comment:
I'm not sure `message_override` is a good choice in naming here, as without looking at the implementation, I'd expect the string I send here completely overrides the message that gets sent. Whereas actually it just adds to it, maybe something like `Message_extended_detail` or something to that affect.

Reply:
Changed in 7dfd5afe